bigquery_etl/shredder/config.py

```diff
@@ -95,14 +92,6 @@ def fields(self) -> Tuple[str, ...]:
     f" UNNEST([{CLIENT_ID}, {IMPRESSION_SRC.field}]) AS `_",
     field="_",
 )
-ECOSYSTEM_CLIENT_ID_HMAC_SRC = DeleteSource(
-    table="account_ecosystem_restricted.ecosystem_client_id_deletion_v1",
-    field=ECOSYSTEM_CLIENT_ID_HASH,
-)
-ECOSYSTEM_CLIENT_ID_SRC = DeleteSource(
-    table="account_ecosystem_restricted.ecosystem_client_id_deletion_v1",
-    field=ECOSYSTEM_CLIENT_ID,
-)
```

Comment:
We are now dropping any remaining AET messages in the pipeline, so we will not be accruing any new entries in the stable tables, and there should be no further need for processing `ecosystem_client_id` as part of deletion requests.